plugins/providers/shield/resource.go

```diff
@@ -0,0 +1,157 @@
+package shield
+
+import (
+	"fmt"
+
+	"github.com/odpf/guardian/domain"
+)
+
+const (
+	ResourceTypeTeam         = "team"
+	ResourceTypeProject      = "project"
+	ResourceTypeOrganization = "organization"
+)
+
+type Metadata struct {
+	Email   string `json:"email" mapstructure:"email"`
+	Privacy string `json:"privacy" mapstructure:"privacy"`
```

Comment:
What would `Privacy` field mean here and is this necessary from guardian context ?

Reply:
It is a metadata of team coming from shield, Kept it as the additional information for team if required to filter out later.